scripts/ci/run-vagrant-tests.sh

```diff
@@ -45,6 +48,22 @@ do
   vagrant ssh agent1 -- sudo journalctl -n 100 -u skydive-agent
 
   vagrant ssh analyzer1 -- curl http://localhost:8082
-  retcode=$?
+
+  if [ "$mode" != "container" ]; then
+  vagrant ssh-config | tail -n +4 > vagrant-ssh-config
+  scp -F vagrant-ssh-config ${GOPATH}/src/github.com/skydive-project/skydive/tests/functionals agent1:
+  ANALYZER1_IP=$(vagrant ssh-config analyzer1 | grep HostName | awk '{print $2}')
+  ANALYZER2_IP=$(vagrant ssh-config analyzer2 | grep HostName | awk '{print $2}')
+
+  vagrant ssh agent1 -c "SKYDIVE_ANALYZERS=\"$ANALYZER1_IP:8082 $ANALYZER2_IP:8082\" sudo -E ./functionals"
+
+  if [ "$DEV_PUSH_AUDIT" = "true" ]; then
+      for a in analyzer1 analyzer2 agent1; do
+          vagrant ssh $a -c 'sudo cat /var/log/audit/audit.log | fpaste'
```

Comment:
Why using fpaste ? It's already in the job output

Reply:
would be less verbose I juste need to post process it, to generate the if and pp files